### PR TITLE
Auto ci ms2snoop

### PR DESCRIPTION
**Pull request by Lain-inrae**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/ms2snoop/MS2snoop.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    --r_threshold $r_threshold
+    --min_number_scan $min_number_scan
+;
+    ]]></command>
+
+    <inputs>
+        <param type="data" name="compound_txt" label="compounds_pos.txt" />
+        <param type="data" name="peaklist_frag_tsv" label="peaklist_fragments.tsv" />
+        <param type="data" name="peaklist_preco_tsv" label="peaklist_precursors.tsv" />
+        <param name="tolmz" type="float" value="0.01" label="MZ Tolerence" />
```
Comment (bgruening) [inline tools/ms2snoop/MS2snoop.xml:32]: Would it make sense to add min or max attributes to the float and integer parameters?

File: tools/ms2snoop/MS2snoop.xml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    --tolrt $tolrt
+    --seuil_ra $seuil_ra
+    --mzdecimal $mzdecimal
+    --r_threshold $r_threshold
+    --min_number_scan $min_number_scan
+;
+    ]]></command>
+
+    <inputs>
+        <param type="data" name="compound_txt" label="compounds_pos.txt" />
```
Comment (bgruening) [inline tools/ms2snoop/MS2snoop.xml:29]: the data parameters are missing a format="" attribute

File: tools/ms2snoop/planemo_test.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (bgruening) [inline tools/ms2snoop/planemo_test.sh:1]: this file is not needed here anymore, this repo has CI testing activated

**Review by bgruening — COMMENTED** [review]
Thanks @Lain-inrae!

A small review is inline in the PR.

File: tools/ms2snoop/MS2snoop.xml (hunk 1 above)
Comment (Lain-inrae) [reply tools/ms2snoop/MS2snoop.xml]: I added min values, but for max values, I'm not sure at all.

File: tools/ms2snoop/MS2snoop.xml [hunk 4]
```diff
@@ -0,0 +1,189 @@
+<tool id="ms2snoop" name="MS2 Snoop" version="1.0.0" profile="21.05">
+    <description>
+        MS1/MS2 spectra and associated adducts extraction and cleaning
+    </description>
+
+    <requirements>
+        <requirement type="package">r-base</requirement>
```
[the remark this answers is not in the document]
Comment (Lain-inrae) [reply tools/ms2snoop/MS2snoop.xml:7]: The versions does not really matter, as the script only uses one package - parseopt - that does not evolve anymore and its API will not change.
Enforcing the version number will have no impact on future result's values.

File: tools/ms2snoop/.gitignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+tool_test_output.*
```
[the remark this answers is not in the document]
Comment (Lain-inrae) [reply tools/ms2snoop/.gitignore:1]: What do you mean?
The files with this name are not present in the folder... and should be ignored from the project as they are results of planemo test.

**Review by fgiacomoni — APPROVED** [review]
Thank you @bgruening for the review and @Lain-inrae for the work on the (too) complex CI/CD task.

File: tools/ms2snoop/.gitignore (hunk 5 above)
[the remark this answers is not in the document]
Comment (lecorguille) [reply tools/ms2snoop/.gitignore:1]: This file is located at the root of the git repository :)

File: tools/ms2snoop/MS2snoop.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (lecorguille) [reply tools/ms2snoop/MS2snoop.xml]: It's still better if one day, R reach the version 5.x or 6.x, you can't be sure that your script is still compatible.

And by the way, it's a good practice for reproducibility

**Pull request merged**